### ISSUE TITLE
New job submission logging is incomplete

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#346]
Originally reported of Google Code with ID 290.

```
When a new job is submitted, this even is indeed logged (_INFO Received new job_), but nothing is logged that reports why job id(s) were allocated, or what the jobs were:

{{{
2012-10-31 11:50:50,632: ganeti-masterd pid=587/ClientReq6 INFO Received new job
}}}

Only later we see in the logs and we can match by client number or similar tricks the job id:

{{{
2012-10-31 12:23:29,846: ganeti-masterd pid=587/ClientReq6 INFO Received job poll request for 962197
}}}
```
Originally added on 2012-10-31 13:02:44 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-31 13:03:05 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in commit 4c91d2ad7a4addc4c6e73f07c27456c9954b927e, will be released in 2.6.2
```
Originally added on 2012-11-01 13:10:57 +0000 UTC.

Added Labels: Type-Enhancement
Changed State: Fixed
Added to Milestone: Release2.6

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking as released the bugs fixed in 2.6.2.
```
Originally added on 2013-01-11 12:13:05 +0000 UTC.

Changed State: Released
